src/components/views/elements/Dropdown.js

```diff
@@ -115,7 +115,7 @@ export default class Dropdown extends React.Component {
 
     componentWillReceiveProps(nextProps) {
         this._reindexChildren(nextProps.children);
-        const firstChild = React.Children.toArray(nextProps.children)[0];
+        const firstChild = nextProps.children[0];
```

Comment:
I have seen the dd get into that state first-hand so I think it'd be better to guard against it. There's nothing to highlight or to index, so it makes sense to return early.